modules/nw-dns-forward.adoc

```diff
@@ -42,7 +42,13 @@ spec:
     - foo.com
     forwardPlugin:
       policy: Random <3>
-      upstreams: <4>
+      transportConfig:
+        tls:
+          caBundle
+            name: mycacert
+          serverName: upstream-tls <4>
```

Comment:
Since most cases we'll expect an FQDN, it might make sense to expand this a little more to be something like `dnstls.example.com`.